### PR TITLE
Update sideloading article

### PR DESCRIPTION
**Pull request by kbrandl**
Add sideloading instructions for the new Outlook on the web. Per Sohail (Outlook PM), "new Outlook on the web" and "classic Outlook on the web" are the current (official) product names for the two flavors of Outlook on Office 365.

### CODE REVIEW

File: docs/add-ins/sideload-outlook-add-ins-for-testing.md [hunk 1]
```diff
@@ -15,17 +15,52 @@ You can use sideloading to install an Outlook add-in for testing without having
...
+> If your organization has included its logo in the mailbox toolbar, you might see something slightly different than shown in the preceding images.
+
+### Sideload an add-in in the new Outlook on the web
+
 1. Go to [Outlook in Office 365](https://outlook.office.com).
 
-1. Choose the gear icon in the top-right section and select **Manage add-ins**.
+1. In Outlook on the web, create a new message or reply to an existing message.   
+
+1. Choose **...** from the bottom of the new message window and then select **Get Add-ins** from the menu that appears.
```
Comment (ElizabethSamuel-MSFT) [inline docs/add-ins/sideload-outlook-add-ins-for-testing.md]: I suggest we remove "window" to account for a reply to an existing message which doesn't necessarily go to a new window.

**Comment by ElizabethSamuel-MSFT** [comment]
Desktop: I recommend we add a screenshot for the "Browse Add-ins" button which replaced the "Store" one in 2019.

File: docs/add-ins/sideload-outlook-add-ins-for-testing.md (hunk 1 above)
Comment (kbrandl) [reply docs/add-ins/sideload-outlook-add-ins-for-testing.md]: Done.

**Comment by jasonjoh** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/images/outlook-on-the-web-classic-toolbar.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-classic-toolbar.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-on-the-web-new-get-add-ins.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-new-get-add-ins.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-on-the-web-new-my-add-ins.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-new-my-add-ins.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-on-the-web-new-toolbar.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-new-toolbar.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-sideload-desktop-store.PNG](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-sideload-desktop-store.PNG) | :white_check_mark:Succeeded | |
[docs/add-ins/sideload-outlook-add-ins-for-testing.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/sideload-outlook-add-ins-for-testing.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C21%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201811211721591874-426%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bDejECLMzuRcuHz0Ai6YKjNP8Y2wstqJPZ37cEos8Fk%3D&st=2018-11-21T17%3A17%3A55Z&se=2018-12-22T17%3A22%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jasonjoh** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/images/outlook-on-the-web-classic-toolbar.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-classic-toolbar.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-on-the-web-new-get-add-ins.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-new-get-add-ins.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-on-the-web-new-my-add-ins.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-new-my-add-ins.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-on-the-web-new-toolbar.png](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-on-the-web-new-toolbar.png) | :white_check_mark:Succeeded | |
[docs/add-ins/images/outlook-sideload-desktop-store.PNG](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/images/outlook-sideload-desktop-store.PNG) | :white_check_mark:Succeeded | |
[docs/add-ins/sideload-outlook-add-ins-for-testing.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-sideloading/docs/add-ins/sideload-outlook-add-ins-for-testing.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C21%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201811211731458084-426%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hD%2BDRsjeCKu9BpVcYyobjNCmCqtz7pVNghPEX%2B0qO1Q%3D&st=2018-11-21T17%3A27%3A46Z&se=2018-12-22T17%3A32%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/sideload-outlook-add-ins-for-testing.md [hunk 2]
```diff
@@ -15,17 +15,52 @@ You can use sideloading to install an Outlook add-in for testing without having
...
+> If your organization has included its logo in the mailbox toolbar, you might see something slightly different than shown in the preceding images.
+
+### Sideload an add-in in the new Outlook on the web
+
 1. Go to [Outlook in Office 365](https://outlook.office.com).
 
-1. Choose the gear icon in the top-right section and select **Manage add-ins**.
+1. In Outlook on the web, create a new message or reply to an existing message.   
+
+1. Choose **...** from the bottom of the new message and then select **Get Add-ins** from the menu that appears.
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/add-ins/sideload-outlook-add-ins-for-testing.md]: And to answer your question: yes -- the "Get Add-ins" option in the menu (regardless of which mode you access it from: Message Compose, Message Read, Appointment Compose, Appointment Read) loads the page where you can "add a custom add-in" (aka sideload).  In the interest of simplicity, Sohail recommended that we not describe the many places where you can access the "Get Add-ins" option (modeled after what he did in his [blog post](https://support.office.com/en-us/article/using-add-ins-in-outlook-on-the-web-8f2ce816-5df4-44a5-958c-f7f9d6dabdce?ui=en-US&rs=en-US&ad=US).

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT thanks for the review and feedback!  I think I've responded to everything; please have another look when you have a chance. Thanks!

**Pull request merged**